src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -28,12 +28,12 @@ protected override async Task<bool> ExecuteAsync(CancellationToken cancellationT
     {
         var loggerWrapper = new LoggerWrapper(this.Log);
         var processWrapper = new ProcessWrapper(this.OpenApiToolsDirectoryPath);
```

Comment:
Not related to this story, but I saw that the process wrapper dumps the process output to the console, however when building a .NET project things go through the MSBuild logger for people to control the verbosity. I find it weird to bypass this and log things straight to the console.

The only scenario where I think it would be okay would be in an Azure DevOps pipeline run. Like, if we detect that we are running in Azure DevOps (there's the corresponding environment variables) then we can print instructions to attach the generated reports to the pipeline run

Reply:
Will address this in future stories